parlai/core/agents.py

```diff
@@ -392,6 +396,18 @@ def get_agent_module(dir_name):
         module_name = 'parlai.agents.legacy_agents.{m}.{m}_v{v}'.format(
             m=model_name, v=s[2])
         class_name = name_to_agent_class(model_name)
+    elif dir_name.startswith('projects:'):
+        # e.g. -m projects:personachat:kvmemnn
+        s = dir_name.split(':')
+        if len(s) != 3:
+            raise RuntimeError('projects paths should follow pattern '
+                               'projects:folder:model; you used {}'
+                               ''.format(dir_name))
+        folder_name = s[1]
+        model_name = s[2]
+        module_name = 'projects.{v}.{m}.{m}'.format(
```

Comment:
let's make it `{p}` instead of `{v}`.

Reply:
Just wondering: does this automatically insert the `parlai.`? It probably should?

Also wondering how this deals with capitalizations